apps/nextjs-website/react-components/components/VideoImage/VideoImage.helpers.tsx

```diff
@@ -137,30 +165,57 @@ export const VideoText = ({
 export const ImageText = ({
   title,
   subtitle,
+  theme = 'dark',
+  themeVariant,
 }: {
   title?: string;
-  subtitle?: string;
+  subtitle?: string | JSX.Element;
```

Comment:
```suggestion
  subtitle?: JSX.Element;
```
Only allow for rendered markdown to be passed.